app/Comment.php

```diff
@@ -7,35 +7,48 @@
 
 class Comment extends BaseModel
 {
-    protected $appends = ["author"];
+    protected $appends = ['author'];
 
-    public function user(){
+    public function user()
+    {
         return $this->belongsTo(User::class);
     }
 
-    public function ticket(){
+    public function ticket()
+    {
         return $this->belongsTo(Ticket::class);
     }
 
-    public function author(){
-        return $this->user ? : $this->ticket->requester;
+    public function author()
+    {
+        return $this->user ?: $this->ticket->requester;
     }
 
-    public function attachments() {
+    public function attachments()
+    {
         return $this->morphMany(Attachment::class, 'attachable');
     }
 
-    public function getAuthorAttribute(){
-        return array_only($this->author()->toArray(),["name","email"]);
+    public function getAuthorAttribute()
+    {
+        return array_only($this->author()->toArray(), ['name', 'email']);
     }
 
-    public function notifyNewComment(){
-        tap(new NewComment($this->ticket, $this), function($newCommentNotification) {
-            if ($this->ticket->team) $this->ticket->team->notify($newCommentNotification);
-            if ($this->ticket->user && (!auth()->user() || auth()->user()->id != $this->ticket->user->id)) $this->ticket->user->notify($newCommentNotification);
-            if ($this->ticket->requester && auth()->user()) $this->ticket->requester->notify($newCommentNotification);
+    public function notifyNewComment()
+    {
+        tap(new NewComment($this->ticket, $this), function ($newCommentNotification) {
+            if ($this->ticket->team) {
+                $this->ticket->team->notify($newCommentNotification);
+            }
+            if ($this->ticket->user && (! auth()->user() || auth()->user()->id != $this->ticket->user->id)) {
```

Comment:
I would recommend to be verbose here. What are you checking? The following snippet, for instance, is easier to understand.

```
if ($this->shouldNotifyUser()) {
    // ...
}
```

Reply:
thanks! new commit applies all these comments, long life to Clean code!